packages/react-ai/.eslintrc.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
-  extends: ['@aws-amplify/amplify-ui/react'],
-  ignorePatterns: ['.eslintrc.js', 'dist', 'rollup.config.ts'],
+  extends: '@aws-amplify/amplify-ui/react',
+  ignorePatterns: ['.eslintrc.js', 'coverage', 'dist', 'node_modules'],
   // point to local tsconfig
   parserOptions: { project: ['tsconfig.json'], tsconfigRootDir: __dirname },
   overrides: [
     {
       extends: ['@aws-amplify/amplify-ui/jest'],
       files: ['**/__mocks__/**', '**/__tests__/**'],
     },
+    { files: '*.mjs' },
```

Comment:
It runs the default set of rules

Reply:
They were not but this could potentially be removed as some changes were made after it was added, either way preference is not to make any changes to this PR unless they impact the bundle output